src/main/java/org/example/AccountType.java

```diff
@@ -1,7 +1,14 @@
 package org.example;
 
 public enum AccountType {
-    BASIC,
-    SAVING,
-    CURRENT
+    BASIC(1),
+    SAVING(2),
```

Comment:
this is the thing i wanted from you to do it,
so enum is not just being general values, but they can data.

Good work...